### ISSUE TITLE
询问

### ISSUE THREAD

**Issue by feitao1993** [hamzamk/ship_detection#1]
请问你的数据是在哪里下载的啊？格式是什么，有点困惑

**Comment by hamzamk** [comment]
Hey, which data are you talking about? and I kept this code as a backup so it needs refinement. 

is it the training data you are talking about?

**Comment by feitao1993** [comment]
Yes, I want to run the process to test the result, so can you share me the complete train data? Thank you very much, my email is 1607516368@qq.com.

**Comment by hamzamk** [comment]
Hey, you can get the data from OpenSARShip website. It is publicly available at
http://opensar.sjtu.edu.cn/ all thanks to the contributors at SJTU. Google it and read their paper I suggest, to understand the dataset.
